src/components/SearchForm/SearchForm.tsx

```diff
@@ -30,9 +31,9 @@ export const SearchForm = () => {
 
         if (!isValidInput(input)) {
             if (submit) {
-                setErrorMsg('errorInputValidationLength');
+                setClientErrorMsg('errorInputValidationLength');
             } else {
-                setErrorMsg(null);
+                setClientErrorMsg(null);
```

Comment:
Bør sette både client og server error til null her (og de to andre stedene den settes til null. Ellers kan den ene bli hengende igjen selv når bruker har skrevet inn noe nytt.

Reply:
Fixed 🔧